tests/playwright/sanity/includes/widgets/playing-cards.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect, test } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorPage from '../../../pages/editor-page';
+import PlayingCardContent from '../../../pages/widgets/playing-cards-page';
+test.describe( 'Playing Cards', () => {
+	test( 'it should display 1 playing card on adding the widget on default', async ( { page }, testInfo ) => {
+		const wpAdmin: WpAdminPage = new WpAdminPage( page, testInfo );
+		const editor: EditorPage = new EditorPage( page, testInfo );
+		// Creating new playing card
+		await wpAdmin.openNewPage();
+		await editor.closeNavigatorIfOpen();
+		await editor.addWidget( 'playing-cards' );
+		await editor.publishAndViewPage();
+		// Waiting for page to load
+		await page.waitForLoadState( 'load' );
+		// Tests
+		const playingCardsWrapperElement = page.locator( '.elementor-playing-cards-wrapper' );
+		const playingCardsElements = page.locator( '.elementor-playing-card' );
+		await expect( playingCardsWrapperElement ).toBeVisible();
+		await expect( playingCardsElements ).toHaveCount( 1 );
+	} );
+
+	test( 'it should be able to add another playing card with the right values', async ( { page }, testInfo ) => {
+		const wpAdmin: WpAdminPage = new WpAdminPage( page, testInfo );
+		const editor: EditorPage = new EditorPage( page, testInfo );
+		const playingCardContent: PlayingCardContent = new PlayingCardContent( page, testInfo );
+		// Creating new playing card
+		await wpAdmin.openNewPage();
+		await editor.closeNavigatorIfOpen();
+		await editor.addWidget( 'playing-cards' );
+		await playingCardContent.addNewCard( 'J', '♦' );
+		await editor.publishAndViewPage();
+		// Waiting for page to load
+		await page.waitForLoadState( 'load' );
+		// Tests
+		const playingCardsWrapperElement = page.locator( '.elementor-playing-cards-wrapper' );
+		const playingCardsElements = page.locator( '.elementor-playing-card' );
+		const firstPlayingCard = page.locator( '.elementor-playing-cards-wrapper' ).getByText( 'A♠' );
+		const secondPlayingCard = page.locator( '.elementor-playing-cards-wrapper' ).getByText( 'J♦' );
+
+		await expect( playingCardsWrapperElement ).toBeVisible();
+		await expect( playingCardsElements ).toHaveCount( 2 );
+		await expect( firstPlayingCard ).toBeVisible();
+		await expect( secondPlayingCard ).toBeVisible();
+		await expect( firstPlayingCard ).toHaveClass( 'elementor-playing-card-value elementor-playing-card-black' );
```

Comment:
CSS Classes rarely change in Elementor once a Widget goes live. But CSS values can change and sometimes regress from time to time and would be a better way to test. It's not mandatory to change on this PR just good to keep it in mind. 

In some cases through like with border & color assertions with CSS values are sometimes slightly off and it's mostly better to test those props with a screenshot.